extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ContainerAdapter.java

```diff
@@ -5,7 +5,6 @@
 import io.dekorate.kubernetes.config.Env;
 import io.dekorate.kubernetes.config.Mount;
 import io.dekorate.kubernetes.config.Port;
-import io.dekorate.kubernetes.decorator.AddEnvVarDecorator;
```

Comment:
LOL - This really through me off regarding where your `AddEnvVarDecorator` was being used :)